pkgs/applications/audio/bespokesynth/default.nix

```diff
@@ -1,40 +1,19 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, fetchzip
-, libjack2, alsa-lib, freetype, libX11, libXrandr, libXinerama, libXext, libXcursor
-, libGL, python3, ncurses, libusb1
-, gtk3, webkitgtk, curl, xvfb-run, makeWrapper
+{ lib,
+  stdenv,
+  fetchFromGitHub, fetchzip, # For vst-sdk
+  cmake, pkg-config, ninja,
+  # List obtained in https://github.com/BespokeSynth/BespokeSynth/blob/main/azure-pipelines.yml
+  libjack2, alsa-lib, alsa-tools,
+  freetype, libX11, libXrandr, libXinerama, libXext, libXcursor, libGL, libusb1,
+  libxcb, xcbutil, libxkbcommon, xcbutilkeysyms, xcb-util-cursor,
+  gtk3, webkitgtk,
+  python3,
+  curl, pcre, mount, gnome, patchelf, makeWrapper,
   # "Debug", or "Release"
-, buildType ? "Release"
+  buildType ? "Release",
```

Comment:
Leading commas is the style preference for Nixpkgs, `nixpkgs-fmt` will also enforce this. Please try running `nixpkgs-fmt` on this file.

Reply:
Thanks. `nixpkgs-fmt` forces me to have everything on a different line, but I think I prefer to group them on the same line by "type" if it's not a problem. But I'll use the leading comma.